extensions/tree-inspector/tree-inspector.js

```diff
@@ -96,6 +104,12 @@ var TreeInspector = (function() {
             .then(function () {
                 if (emphasisNodePtr != null) {
                     that.emphasizeNode(emphasisNodePtr, that.treeRoot, that.treeReader);
+                }                
+            })
+            .then(function () {
+                if (that.refreshOnBreak && that.refreshOnBreakAndContinueEnabled)

```

Comment:
Do we need to check for "refreshOnBreak" here? Won't "refreshOnBreakAndContinueEnabled" only be true if "refreshOnBreak" is true?

Reply:
I dont want UI to control app logic, the check isnt harmful. Ill let it be.